### PR TITLE
Add `is-start_selection` and `is-end_selection` modifiers for a day

### PR DESCRIPTION
**Pull request by PeterMK85**
## Changeset:
+ Able to provide alternative style for the first and the last element of a selection 
  _(Add `is-start_selection` and `is-end_selection` modifiers for a day)_
+ Covered with UT
+ Increased version number

## Screenshot:
image

### CODE REVIEW

File: src/calendar/week.jsx [hunk 1]
```diff
@@ -61,11 +61,14 @@ export default class Week extends React.Component {
...
 
   _dateClasses(date) {
-    const {today, activeMonth} = this.props
+    const {today, activeMonth, selectedMax, selectedMin} = this.props
+
     return classnames({
       'is-selected': this._dateSelected(date),
       'is-today': isSameDay(today, date),
       'is-current_month': isSameMonth(date, activeMonth),
+      'is-start_selection': isSameDay(selectedMin, date),
```
Comment (dmaklygin) [inline src/calendar/week.jsx]: is `selectedMin`  the necessary property?

**Comment by dmaklygin** [comment]
👍

**Comment by havenchyk** [comment]
👍

File: src/calendar/week.jsx (hunk 1 above)
Comment (kossnocorp) [inline src/calendar/week.jsx]: No, it's not `isRequired` (see prop types), so you must do `selectedMin && isSameDay(selectedMin, date)` and add a test case for that.

The same for `selectedMax`.

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "git+https://github.com/toptal/simple-react-calendar.git"
   },
-  "version": "1.2.2",
+  "version": "1.2.3",
```
Comment (kossnocorp) [inline package.json]: It's not a bug fix, so it should be `1.3.0`.

**Pull request merged**